README.md

```diff
@@ -102,6 +103,15 @@ These fields are merged with all library, executable, test, and benchmark compon
 | `reexported-modules` | · | | |
 | | `default-language` | `Haskell2010` | |
 
+#### <a name="internal-library-fields"></a>Internal library fields
+
+| Hpack | Cabal | Default | Notes |
+| --- | --- | --- | --- |
+| `exposed-modules` | · | All modules in `source-dirs` less `other-modules` | |
+| `other-modules` | · | All modules in `source-dirs` less `exposed-modules` | |
+| `reexported-modules` | · | | |
+| | `default-language` | `Haskell2010` | |
+
```

Comment:
Given the other changes, we can get rid of this too and reference `library-fields` instead.